src/coreclr/src/jit/lowerarmarch.cpp

```diff
@@ -1464,6 +1464,17 @@ void Lowering::ContainCheckHWIntrinsic(GenTreeHWIntrinsic* node)
                         MakeSrcContained(node, intrin.op3);
                     }
                 }
+                else if (intrin.op3->OperIs(GT_HWINTRINSIC))
+                {
+                    const HWIntrinsic hwIntrin(intrin.op3->AsHWIntrinsic());
```

Comment:
Can this be named `op3` or `data` instead of `hwIntrin`?

Reply:
Sure.